.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Pull Request Template
+
+This PR closes the issues: #...
+
+### Description
+
+Here provide a list **_all_** new changes of this PR (new files, new utils, bug fix, etc).
+
+### Checklist:
+
+- [ ] I have followed [Contributor Guidelines](https://github.com/nestauk/nesta_ds_utils/blob/dev/CONTRIBUTING.md)
+- [ ] I have ran and passed all tests locally with `>> pytest`
```

Comment:
Maybe add a line for "I have updated the documentation to include any new functionality I have added or modified" and "I have written unit tests for any new functionality I have added". I know that's in the contributor guidelines but doesn't hurt to add it here as well?

Reply:
Yes, good point! I actually included these originally and then removed them because overlapping the guidelines.. but I think it doesn't hurt in repeat them here.